scripts/policies/data.json

```diff
@@ -0,0 +1,14 @@
+{
+    "classes": {
+        "default": {
+            "pki": 1,
+            "tls": 1,
+            "tpm": 1
+        },
+        "production": {
+            "pki": 2,
+            "tls": 2,
+            "tpm": 1
+        }
+    }
+}
```

Comment:
Should we add support for the `pki-http` annotator now or wait till it's integrated in the Java, Python SDKs and the example apps? If so we might want to update the `config-mqtt` files in the `calculator` to include them also.

Reply:
This should be handled as a follow-up issue once the repo is populated.